src/org/jitsi/impl/neomedia/transform/srtp/BaseSRTPCryptoContext.java

```diff
@@ -268,7 +268,7 @@ protected BaseSRTPCryptoContext(
      * @param pkt the RTP packet to be authenticated
      * @param rocIn Roll-Over-Counter
      */
-    synchronized protected void authenticatePacketHMAC(RawPacket pkt, int rocIn)
+    protected void authenticatePacketHMAC(RawPacket pkt, int rocIn)
```

Comment:
This should definitly not be here (git commit -a ...), thanks for the catch, I'll remove it